packages/docs-ui/README.md

```diff
@@ -54,5 +55,22 @@ import { MenuButton } from '@braid-design-system/docs-ui';
 | open    | `boolean`    | The Menu can either be open or closed. If open, the button will change to a close icon (defaults to `false`). |
 | onClick | `() => void` | A callback function to handle button presses (defaults to `false`).                                           |
 
+### `SideNavigationGroup`
+
+A group of SideNavigationItems, with an optional title.
+
+```tsx
+import { SideNavigationGroup } from '@braid-design-system/docs-ui';
+
+<SideNavigationGroup title={title} items={items} />;
+```
+
+#### Props
+
+| props | value                 | description                                                             |
+| ----- | --------------------- | ----------------------------------------------------------------------- |
+| title | `string`              | Optional title of the Navigation Group.                                 |
+| items | `SubNavigationItem[]' | An array of items in the sidebar, each linking to a specific docs page. |
```

Comment:
Given this isnt exposed, have been playing with how we might show the structure here. We can play with this offline.